opensearch/chart/values.yaml

```
@@ -1,19 +1,51 @@
 # SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
 # SPDX-License-Identifier: Apache-2.0
 
-global:
-  ## common greenhouse values to use
-  ##
-  greenhouse:
-    # base domain of the greenhouse organisation
-    baseDomain:
-  certManager:
-    enable: true
-    issuerRef:
+# -- Enable cert-manager integration for issuing TLS certificates
+certManager:
+  enable: true
```

this will make the README more readable

```suggestion

certManager:
  # -- Enable cert-manager integration for issuing TLS certificates
  enable: true
```

---

Thanks, good point! :)